tests/functional/multitenant/apns.rs

```diff
@@ -0,0 +1,118 @@
+use {
+    crate::context::EchoServerContext,
+    echo_server::handlers::create_tenant::TenantRegisterBody,
+    random_string::generate,
+    test_context::test_context,
+};
+
+// #[test_context(EchoServerContext)]
+// #[tokio::test]
+// async fn tenant_update_apns(ctx: &mut EchoServerContext) {
+//     let charset = "1234567890";
+//     let random_tenant_id = generate(12, charset);
+//     let payload = TenantRegisterBody {
+//         id: random_tenant_id.clone(),
+//     };
+//
+//     // Register tenant
+//     let client = reqwest::Client::new();
+//     let response = client
+//         .post(format!("http://{}/tenants", ctx.server.public_addr))
+//         .json(&payload)
+//         .send()
+//         .await
+//         .expect("Call failed");
+//
+//     // Send valid token/cert
+//     // TODO figure out how to get valid creds into test!
+//     let api_key = env!("ECHO_TEST_FCM_KEY");
+//     let form = reqwest::multipart::Form::new().text("api_key", api_key);
+//
+//     let response = client
+//         .post(format!(
+//             "http://{}/tenants/{}/apns",
+//             ctx.server.public_addr, &random_tenant_id
+//         ))
+//         .multipart(&form)
+//         .send()
+//         .await
+//         .expect("Call failed");
+//
+//     assert!(
+//         response.status().is_success(),
+//         "Response was not successful"
+//     );
+// }
```

Comment:
is this a leftover?

Reply:
It needs to be added back once we have a good way to pass valid APNS credentials into the tests